package NYT.NTransactionClient.NProto;

////////////////////////////////////////////////////////////////////////////////

message TTransactionActionData
{
    required string action_type = 1;
    required bytes action_value = 2;
}

////////////////////////////////////////////////////////////////////////////////
